A few notifications.

Anonymioo opened the pull request:
Added 4 Notifications.
Not sure about last one.

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_pinterest",
```
As of #4375, adding a consistent label format for these.
```suggestion
            eventlabel="mas_wrs_pinterest",
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+        code="WRS"
+    )
+
+label notif_pinterest
```
```suggestion
label mas_wrs_pinterest
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('monika_pinterest')
```
```suggestion
        $ mas_unlockFailedWRS('mas_wrs_pinterest')
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_duolingo",
```
```suggestion
            eventlabel="mas_wrs_duolingo",
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+        code="WRS"
+    )
+
+label notif_duolingo
```
```suggestion
label mas_wrs_duolingo
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('notif_duolingo')
```
```suggestion
        $ mas_unlockFailedWRS('mas_wrs_duolingo')
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_wikipedia",
```
```suggestion
            eventlabel="mas_wrs_wikipedia",
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+        code="WRS"
+    )
+
+label notif_wikipedia
```
```suggestion
label mas_wrs_wikipedia
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('notif_wikipedia')
```
```suggestion
        $ mas_unlockFailedWRS('mas_wrs_wikipedia')
```

Anonymioo commented:
Wrong branch.

multimokia commented:
FYI, you can rebase without closing

multimokia commented:
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-put-my-pull-request-in-the-wrong-branch-and-dont-want-to-suffer-the-wrath-of-devs-how-do-i-fix-it

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="reddit_rule34",
```
One thing about the other one was that it was supposed to be *specifically* for Monika related searches, and not general. The expressions for the `show`s here don't really work if you're potentially looking at one of the others.

Anonymioo replied:
> One thing about the other one was that it was supposed to be specifically for Monika related searches, and not general. The expressions for the shows here don't really work if you're potentially looking at one of the others.

I got it. I just tried to make it for Reddit version.

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+        code="WRS"
+    )
+
+label mas_wrs_pinterest
```
```suggestion
label mas_wrs_pinterest:
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+        code="WRS"
+    )
+
+label mas_wrs_duolingo
```
```suggestion
label mas_wrs_duolingo:
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
+        code="WRS"
+    )
+
+label mas_wrs_wikipedia
```
```suggestion
label mas_wrs_wikipedia:
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,66 @@ label display_notif(title, body, group=None, skip_checks=False):
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_pinterest",
+            category=['pinterest'],
+            rules={"skip alert": None},
```
```suggestion
            rules={"notif-group": "Window Reactions", "skip alert": None},
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,66 @@ label display_notif(title, body, group=None, skip_checks=False):
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_duolingo",
+            category=['duolingo'],
+            rules={"skip alert": None},
```
```suggestion
            rules={"notif-group": "Window Reactions", "skip alert": None},
```

multimokia commented on Monika After Story/game/zz_windowreacts.rpy:
```diff
@@ -320,6 +321,66 @@ label display_notif(title, body, group=None, skip_checks=False):
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_wikipedia",
+            category=['wikipedia'],
+            rules={"skip alert": None},
```
```suggestion
            rules={"notif-group": "Window Reactions", "skip alert": None},
```

multimokia commented:
Putting don't merge on this until #4398 gets in

multimokia commented:
Okay. Will need you to update to the new usage, and possibly add some more variants to the responses since now you can just feed the notif function a list of things you want. (one will be picked at random)

Anonymioo commented:
> Okay. Will need you to update to the new usage, and possibly add some more variants to the responses since now you can just feed the notif function a list of things you want. (one will be picked at random)

Updated. Added some variants.

Anonymioo commented:
@multimokia 
I found typo. Or it's normal?
`"Learning new ways to say 'I love you,' [player]?",`

multimokia commented:
That's normal. Punctuation goes inside quotations.

Anonymioo commented:
> That's normal. Punctuation goes inside quotations.

Pardon. Didn't knew.

multimokia commented:
No worries
